lib/screens/alerts/alert.ex

```diff
@@ -289,6 +292,7 @@ defmodule Screens.Alerts.Alert do
     end
   end
 
+  # This was only used for the AlertPriorityController. What was that? Is it still used?
```

Comment:
I've actually never heard of that 🤔 It still has a route but it isn't being used in the frontend. It's also v1 only. Might be able to remove it when/if we cleanup v1.

Reply:
I'm one of the contributors to this file in the blame, but...
![image](https://user-images.githubusercontent.com/63608771/234082855-4f9dd03e-d2a0-442e-be7d-fe182d6a813a.png)

Reply:
I'd say it's safe to remove!

Reply:
I'm looking back at my comments and seeing they were a little disorienting -- like I left the question "what is this", but elsewhere noted "used here". Lol, that's unhelpful. I'll be trying to clean all this up. Christian is correct, this is needed for v1, so I'll label it as such, and we can remove it when v1 is retired!